ibis/tests/all/test_window.py

```diff
@@ -19,6 +19,7 @@
             lambda t, win: t.float_col.lead().over(win),
             lambda t: t.float_col.shift(-1),
             id='lead',
+            marks=pytest.mark.xfail_backends((MapD,)),
```

Comment:
not sure why but it was working before ... now it raises error `Series values are different (0.27397 %)` it seems it is related to so nan values ... I need more time to understand this difference and I can fix that in a follow up PR (I will add win ops)

Reply:
Sounds good to me!